frontend/utilities/helpers.ts

```diff
@@ -590,43 +590,66 @@ export const humanHostLastRestart = (
   detailUpdatedAt: string,
   uptime: number
 ): string => {
-  const currentDate = new Date();
-  const updatedDate = new Date(detailUpdatedAt);
-  const millisecondsLastUpdated = currentDate.getTime() - updatedDate.getTime();
+  if (
```

Comment:
We should probably use a different pattern that tries to parse the date first before the conditionals. This occurs in a number of places so I'd suggest making separate ticket to revisit how dates are being used throughout the app.